### PR TITLE
Add release notes for 2.7.2

### PR DESCRIPTION
**Pull request by madkinsz**
[Rendered](https://github.com/PrefectHQ/prefect/blob/release-notes-2.7.2/RELEASE-NOTES.md)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+- Fix path argument behavior in `LocalFileSystem` block - https://github.com/PrefectHQ/prefect/pull/7891
+
+### Documentation
+- Add CSS for beta and experimental badges — https://github.com/PrefectHQ/prefect/pull/7831
+- Add documentation for Automations UI — https://github.com/PrefectHQ/prefect/pull/7833
+- Mention recipes and tutorials under Recipes and Collections pages — https://github.com/PrefectHQ/prefect/pull/7876
+- Add cloud icons and admonitions — https://github.com/PrefectHQ/prefect/pull/7878
+- Add documentation for Task Run Concurrency UI — https://github.com/PrefectHQ/prefect/pull/7840
+- Add `with_options` example to collections usage docs — https://github.com/PrefectHQ/prefect/pull/7894
+- Add link to orion design and better title to UI readme — https://github.com/PrefectHQ/prefect/pull/7484
```
Comment (anna-geller) [inline RELEASE-NOTES.md:31]:
```suggestion
- Add a link to orion design and better title to UI readme — https://github.com/PrefectHQ/prefect/pull/7484
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+- Add Opsgenie notification block — https://github.com/PrefectHQ/prefect/pull/7778
+- Add individual concurrency limit page with active runs list — https://github.com/PrefectHQ/prefect/pull/7848
+- Add `PREFECT_KUBERNETES_CLUSTER_UID` to allow bypass of `kube-system` namespace read — https://github.com/PrefectHQ/prefect/pull/7864
+- Refactor `pause_flow_run` for consistency with engine state handling — https://github.com/PrefectHQ/prefect/pull/7857
+- API: Allow `reject_transition` to return current state — https://github.com/PrefectHQ/prefect/pull/7830
+
+### Fixes
+- Fix bug where agent concurrency slots may not be released — https://github.com/PrefectHQ/prefect/pull/7845
+- Fix circular imports in the `orchestration` module — https://github.com/PrefectHQ/prefect/pull/7883
+- Fix deployments builds with scripts that contain flow calls - https://github.com/PrefectHQ/prefect/pull/7817
```
Comment (anna-geller) [inline RELEASE-NOTES.md:21]:
```suggestion
- Fix deployment builds with scripts that contain flow calls - https://github.com/PrefectHQ/prefect/pull/7817
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
 # Prefect Release Notes
 
+## Release 2.7.2
+
+### Extensions to the new pausing feature!
+
+
+See https://github.com/PrefectHQ/prefect/pull/7738 and https://github.com/PrefectHQ/prefect/pull/7863 for details
```
Comment (anna-geller) [inline RELEASE-NOTES.md:8]: Super optional: We could add more info before referencing the PRs.
```suggestion
This release extends the recently added pause/resume functionality with non-blocking functionality. You can now pass the `reschedule` argument to the `pause_flow_run` utility to gracefully stop the infrastructure process and put the flow run back into a `Scheduled` state. Similarly, you can pause the run from the friendly _Pause_ button directly from the flow run UI page. See https://github.com/PrefectHQ/prefect/pull/7738 and https://github.com/PrefectHQ/prefect/pull/7863 for details
```

**Review by anna-geller — APPROVED** [review]
Minor additions, LGTM

File: RELEASE-NOTES.md (hunk 3 above)
Comment (madkinsz) [reply RELEASE-NOTES.md:8]: @anticorrelator is going to add a full blurb here.

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,71 @@
...
+- API: Allow `reject_transition` to return current state — https://github.com/PrefectHQ/prefect/pull/7830
+
+### Fixes
+- Fix bug where agent concurrency slots may not be released — https://github.com/PrefectHQ/prefect/pull/7845
+- Fix circular imports in the `orchestration` module — https://github.com/PrefectHQ/prefect/pull/7883
+- Fix deployments builds with scripts that contain flow calls - https://github.com/PrefectHQ/prefect/pull/7817
+- Fix path argument behavior in `LocalFileSystem` block - https://github.com/PrefectHQ/prefect/pull/7891
+
+### Documentation
+- Add CSS for beta and experimental badges — https://github.com/PrefectHQ/prefect/pull/7831
```
Comment (tpdorsey) [inline RELEASE-NOTES.md:56]:
```suggestion
```

Comment (tpdorsey) [reply RELEASE-NOTES.md:56]: This is kind of inside baseball and I don't think it rates inclusion in the release notes, but open to other opinions.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,71 @@
...
+- Add documentation for Automations UI — https://github.com/PrefectHQ/prefect/pull/7833
+- Mention recipes and tutorials under Recipes and Collections pages — https://github.com/PrefectHQ/prefect/pull/7876
+- Add cloud icons and admonitions — https://github.com/PrefectHQ/prefect/pull/7878
```
Comment (tpdorsey) [inline RELEASE-NOTES.md:59]:
```suggestion
```

Comment (tpdorsey) [reply RELEASE-NOTES.md:59]: As above, I think we can leave this out.

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,72 @@
 # Prefect Release Notes
 
+## Release 2.7.2
+
+### Extensions to the new pausing feature!
+
+Pausing flow runs with Prefect have become much more powerful!
```
Comment (tpdorsey) [inline RELEASE-NOTES.md:7]:
```suggestion
Pausing flow runs with Prefect has become much more powerful!
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,72 @@
...
+    await marvin_setup()
+    await pause_flow_run(timeout=600, reschedule=True)  # pauses for 10 minutes
+    await marvin_punchline()
+```
+
+If set up as a deployment, running this flow will set up a joke, then pause and leave execution until it is resumed. Once resumed either with the `resume_flow_run` utility or the Prefect UI, the flow will be rescheduled and deliver the punchline.
+
+Deployed flows can also be paused outside of the flow process!
+
+The UI now features a **Pause** button on deployed flows that will stop execution at the beginning of the **next** task that runs. Any currently running tasks will be allowed to complete. Resuming this flow will reschedule it.
```
Comment (tpdorsey) [inline RELEASE-NOTES.md:33]:
```suggestion
The UI now features a **Pause** button on deployed flows that will stop execution at the beginning of the _next_ task that runs. Any currently running tasks will be allowed to complete. Resuming this flow will reschedule it.
```

**Review by tpdorsey — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,81 @@
 # Prefect Release Notes
 
+## Release 2.7.2
+
+### Extensions to the new pausing feature!
+
+Pausing flow runs with Prefect has become much more powerful!
+
+Flow runs can now be paused without blocking the flow run process by passing `reschedule=True` to the `pause_flow_run` function. This feature can free up agents to run other flows while waiting for approval. In order to use rescheduling pauses, flows must be deployed and results must be configured with the `persist_result` option.
```
Comment (madkinsz) [inline RELEASE-NOTES.md:9]: nit: This doesn't free up agents. This frees up infrastructure though. Maybe focus on cost reduction?

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,81 @@
...
+
+```
```
Comment (madkinsz) [inline RELEASE-NOTES.md:11]:
```suggestion
```python
```

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,81 @@
...
+@flow(persist_result=True)
+async def inspiring_joke():
+    await marvin_setup()
+    await pause_flow_run(timeout=600, reschedule=True)  # pauses for 10 minutes
+    await marvin_punchline()
+```
+
+If set up as a deployment, running this flow will set up a joke, then pause and leave execution until it is resumed. Once resumed either with the `resume_flow_run` utility or the Prefect UI, the flow will be rescheduled and deliver the punchline.
+
+Deployed flows can also be paused outside of the flow process!
```
Comment (madkinsz) [inline RELEASE-NOTES.md:31]: This could probably use a bit more details.

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,79 @@
...
+### Documentation
+- Add documentation for Automations UI — https://github.com/PrefectHQ/prefect/pull/7833
+- Mention recipes and tutorials under Recipes and Collections pages — https://github.com/PrefectHQ/prefect/pull/7876
+- Add documentation for Task Run Concurrency UI — https://github.com/PrefectHQ/prefect/pull/7840
+- Add `with_options` example to collections usage docs — https://github.com/PrefectHQ/prefect/pull/7894
+- Add a link to orion design and better title to UI readme — https://github.com/PrefectHQ/prefect/pull/7484
+
+### Collections
+- Add new [`prefect-kubernetes`](https://prefecthq.github.io/prefect-kubernetes/) collection for [Kubernetes](https://kubernetes.io/) — https://github.com/PrefectHQ/prefect/pull/7907
+- Add new [`prefect-bitbucket`](https://prefecthq.github.io/prefect-bitbucket/) collection for [Bitbucket](https://bitbucket.org/product) — https://github.com/PrefectHQ/prefect/pull/7907
```
Comment (madkinsz) [inline RELEASE-NOTES.md:70]:
```suggestion
### Collections
- New [`prefect-kubernetes`](https://prefecthq.github.io/prefect-kubernetes/) collection for [Kubernetes](https://kubernetes.io/) — https://github.com/PrefectHQ/prefect/pull/7907
- New [`prefect-bitbucket`](https://prefecthq.github.io/prefect-bitbucket/) collection for [Bitbucket](https://bitbucket.org/product) — https://github.com/PrefectHQ/prefect/pull/7907
```

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -1,5 +1,85 @@
...
+
+The UI features a **Pause** button for flow runs that will stop execution at the beginning of the _next_ task that runs. Any currently running tasks will be allowed to complete. Resuming this flow will schedule it to start again.
+
+You can also pause a flow run from code: the `pause_flow_run` utility now accepts an optional `flow_run_id` argument. For example, you can pause a flow run from another flow run!
+
+Read the [pause documentation](https://docs.prefect.io/concepts/flows/#pause-a-flow-run) or see the [pull request](https://github.com/PrefectHQ/prefect/pull/7863) for details.
+
+## Pages for individual task run concurrency limits
+
+When viewing task run concurrency in the UI each limit has its own page. Included in the details for each limit is the tasks which are actively part of that limit.
```
Comment (tpdorsey) [inline RELEASE-NOTES.md:47]:
```suggestion
When viewing task run concurrency in the UI, each limit has its own page. Included in the details for each limit is the tasks that are actively part of that limit.
```

**Review by tpdorsey — APPROVED** [review]
Minor suggested edit

**Pull request merged**